rustfmt.toml

```diff
@@ -28,7 +28,7 @@ combine_control_expr = true
 struct_field_align_threshold = 0
 match_arm_blocks = true
 force_multiline_blocks = false
-fn_args_density = "Tall"
+fn_args_layout = "Tall"
```

Comment:
Looks innocuous but not sure why this change is in this PR.

Reply:
Minor rust version update leftovers.